mantidproject/mantid
# Add log message window to workbench

Description of work.

**To test:**

* Build the branch
* Start the workbench and see that the splash screen appears. After a second the window will open and the show the log message widget. It should display the start up messages from `mantid`
* It currently fills the whole window as there are no other widgets available to populate it.

Refs #21251

**Release Notes** 

*Does not need to be in the release notes. Workbench changes will be announced separately.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
